src/container.tsx

```diff
@@ -49,7 +49,10 @@ export default class StickyContainer extends React.Component<Props, State> {
   }
 
   public refresh() {
-    this.onScrollHandler()
+    // refresh might be used when list of items is filtered. we need to
+    // reset scroll position to recalculate properly properties of sticky element.
+    // this will also trigger onScrollHandler
+    this.refs.container.scrollTop = 0
```

Comment:
@edmundas-ramanauskas I don't understand the change. `#refresh()` is used to "simulate a scroll event" and recalculate the current sticky header, in case the layout has changed for any reason but no scroll event has been triggered. But it shouldn't change the scroll position however. Is it really the same method you need ?